java/change-notes/2021-04-06-ssrf-query.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* The query "Server-Side Request Forgery (SSRF)" (`java/ssrf`) has been promoted from experimental to the main query pack. Its results will now appear by default. This query was originally [submitted as an experimental query by @porcupineyhairs](https://github.com/github/codeql/pull/3454).
```

Comment:
The casing here does not match the query name.

Reply:
Done